src/main/java/com/ksm/wstbackoffice/entity/Address.java

```diff
@@ -1,17 +1,38 @@
 package com.ksm.wstbackoffice.entity;
 
 import lombok.Data;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
 
+@Entity
 @Data
 public class Address {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private long id;
+    @ManyToOne(optional=false, cascade= CascadeType.ALL)
```

Comment:
1. What optional=false means?
2. What cascade All means?

Reply:
What optional=false means?
The optional parameter tells JPA whether the value in this field is required or not.
By default optional=true, because I removed parametr optional 

What cascade All means?
Cascade.ALL propagates all operations — including Hibernate-specific ones — from a parent to a child entity.
I changed Cascade.ALL to CascadeType.PERSIST
When we save the Address entity, the Country entity will also get saved.
If will remove the Address entity, the Country entity will not be delete.